include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -71,5 +79,3 @@ static const char* const kOrtSessionOptionsConfigAllowIntraOpSpinning = "session
 // If not specified, the default set of stop ops is used. To specify an empty stop ops types list and disable stop op
 // exclusion, set the value to "".
 static const char* const kOrtSessionOptionsConfigNnapiEpPartitioningStopOps = "ep.nnapi.partitioning_stop_ops";
-
-// NNAPI EP keys end
```

Comment:
if removing the "end" comment, can you also remove the "begin" one and change the note to say "The NNAPI EP options..." instead?

Reply:
Will do in my next change